assets/js/base/components/loading-mask/index.tsx

```diff
@@ -11,14 +10,21 @@ import classNames from 'classnames';
 import './style.scss';
 import Spinner from '../spinner';
 
+interface LoadingMaskProps {
+	children?: React.ReactChildren;
```

Comment:
I see in other places in the codebase we are using `ReactNode` or `ReactNode[]` rather than `React.ReactChildren; `. Do you know which is best practice, and can we decide on a standard to use throughout?